apps/boilerplate-front-end/src/components/header/components/OktaAuth.jsx

```diff
@@ -0,0 +1,272 @@
+import { OktaLogo } from '@/assets/svg/SvgIndex'
+import { useOktaAuth } from '@okta/okta-react'
+import { useEffect, useState } from 'react'
+
+const OktaAuth = () => {
+  const { authState, oktaAuth } = useOktaAuth()
+  const handleLogin = () => oktaAuth.signInWithRedirect()
+  // To use for a Logout function
+  // const handleLogout = () => oktaAuth.signOut()
```

Comment:
are you not supporting log out?

Reply:
It's difficult we have to see with security because of CORS Policy, so I removed it bercause I thought that it's not only a Logout of the BP it's a Logout of OKTA and I'm not sure someone want to be logout of OKTA. So I loooked at `implementation-explorer` app made by SE's they used OKTA Auth but they removed Logo out too. It's possible to do it but it will be like you'll decide.

Reply:
aha ok thanks that is clear!